tasks/christmas-shop/christmas-shop.md

```diff
@@ -0,0 +1,132 @@
+- Using JavaScript to implement the functionality specified in the task.
+
+## Project Work Stages
+
+The task consists of three parts:
+
+- [Christmas Shop. Part-1: Fixed Layout](christmas-shop-part1.md)
+  - In this part of the task, you need to create the 'Home' and 'Gifts' pages based on the layout, which should display correctly when the window width is at least 1440px;
+  - The validity of the work is checked, as well as its compliance with the layout.
+- [Christmas Shop. Part-2: Responsive Design](christmas-shop-part2.md)
+  - In this part of the task, it is necessary to add responsiveness to the pages created in the first stage, up to and including a width of 380px;
+  - The validity of the work is checked, its alignment with the layout at the corresponding page width, layout responsiveness, and the absence of horizontal scroll bars.
+- [Christmas Shop. Part-3: Adding Functionality](christmas-shop-part3.md)
+  - In this part of the task, we use JavaScript to add interactivity to the pages;
+  - The implemented functionality is being checked.
+
+The duration of each part of the task is 1 week.  
+The form of evaluation for each part of the task is a cross-check review.
+
+[Design in Figma](https://www.figma.com/design/zTB01BwWZVoXYK5atH3eZT/Cristmas-Shop)  
+[Graphic materials](img-compressed.zip)
+
+## Creating your own copy of the layout
+
+Start the task by creating your own copy of the layout. To do this:
+
+- Log in to [Figma](https://www.figma.com/);
+- Open the layout;
+- On the left panel, click on the arrow next to the layout name, and select the option "Duplicate to your drafts";
+- At the top left, open the settings, and choose "Back to files";
+- Open the copy of the layout with the label "Drafts".
+
+## Code quality recommendations
+
+Recommendations are provided for reference; strict adherence to them is not expected and will not be checked.
+
+- [Guideline](https://codeguide.academy/html-css.html#html)
+- Code quality guide
+  - [General principles](../../stage1/modules/clean-code/materials/generic-principles.md)
+  - [HTML and CSS recommendations - beginner level](../../stage1/modules/clean-code/materials/html-and-css.md)
+  - [HTML and CSS recommendations - advanced level](../../stage1/modules/clean-code/materials/html-and-css-extended.md)
+
+## Technical requirements
+
+1. The layout is valid, semantic, and matches the design.
+2. The application is displayed correctly and functions properly in the latest version of Google Chrome.
+3. Using HTML preprocessors and template engines (e.g. `Pug`) **is not allowed**.
+4. Using CSS frameworks (e.g., `Bootstrap`) **is not allowed**.
+5. Using JS frameworks (e.g., `Angular`, `React`, `Vue`, etc.) **is not allowed**.
+6. Using outdated libraries (e.g. `JQuery`, etc.) or pre-built libraries (e.g. `Swiper`, etc) to implement functionality **is not allowed**.
+7. Using `TypeScript` **is not allowed**.
+8. Using CSS preprocessors (`SASS`, `SCSS`), `normalize.css` is allowed.
```

Comment:
Could we replace `normalize.css` with `modern-normalize`? It seems more relevant considering the latest `normalize.css` updates were about 6 years ago 🙂 

```suggestion
8. Using CSS preprocessors (`SASS`, `SCSS`), `[modern-normalize](https://github.com/sindresorhus/modern-normalize)` is allowed.
```